src/modules/45-dashboards/components/DashboardCard/__tests__/DashboardCard.test.tsx

```diff
@@ -61,8 +77,10 @@ describe('DashboardCard', () => {
   const cloneText: StringKeys = 'projectCard.clone'
   const deleteText: StringKeys = 'delete'
   const editText: StringKeys = 'edit'
+
   beforeEach(() => {
     jest.clearAllMocks()
+    jest.spyOn(customDashboardServices, 'useGetFolders').mockImplementation(() => ({ data: {} } as any))
```

Comment:
Fine as-is, but you can simplify with the `mockReturnValue` method
```suggestion
    jest.spyOn(customDashboardServices, 'useGetFolders').mockReturnValue({ data: {} } as any)
```